01.w_Defects/pow_related_errors.c

```diff
@@ -20,6 +20,9 @@
 * Complexity: Very high values of number and exponent results in overflow of the answer
 */
 
+// TODO: fix many issues in this file. Notably, some exponents seem to not make much sense, and
```

Comment:
10 ^ 36 == 46. There are also a number of exponents in this file that seem really strange, like 10^3700 which will overflow a double. I think this file needs to be gone over much more thoroughly with a calculator and examination of float.h.